docs/docs/legal/data-processing-agreement.md

```diff
@@ -0,0 +1,229 @@
+      <p>Public Keys: Users who use an authentication procedure with cryptographic keys.</p>
+      <p> External login provider identifiers: Users who use an external login provider.</p>
+      <p>Phone number: Users who use authentication methods with SMS</p>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Profile data</strong></td>
+    <td>
+      <ul>
+        <li>Profile pictures</li>
+        <li>Gender</li>
+        <li>Birthday</li>
+        <li>Language</li>
+        <li>Address(es)</li>
+        <li>Phone number(s)</li>
+      </ul>
+    </td>
+    <td>Users who voluntarily add profile data</td>
+  </tr>
+  <tr>
+    <td><strong>Communication data</strong></td>
+    <td>
+      <ul>
+        <li>Emails</li>
+        <li>Chats</li>
+        <li>Call metadata</li>
+      </ul>
+    </td>
+    <td>Customers and users who communicate with us directly (e.g. support)</td>
+  </tr>
+  <tr>
+    <td><strong>Payment data</strong></td>
+    <td>
+      <ul>
+        <li>Billing address</li>
+        <li>Customer number</li>
+        <li>Customer history</li>
+        <li>Credit rating information</li>
+      </ul>
+    </td>
+    <td>
+      <p>Customers who use services that require payment</p>
+      <p>Credit rating information: Only customers who pay by invoice</p>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Data traffic data</strong></td>
+    <td>
+      <ul>
+        <li>User agent</li>
+        <li>IP addresses</li>
+        <li>Operating system</li>
+        <li>Time and date</li>
+        <li>URL</li>
+        <li>Referrer URL</li>
+        <li>Accept Language</li>
+      </ul>
+    </td>
+    <td>All users</td>
+  </tr>  
+</table>
+
+## Scope and responsibility
+
+Under the Agreement, the Processor shall process Personal Data on behalf of the Customer.
+
+This Annex applies to all processing of Customer&#39;s data (including data of the users of Customer&#39;s organization) with reference to persons (&quot; **Personal Data**&quot;) which is related to the Agreement and which is carried out by the Processor, its employees or agents.
+
+The Customer shall be responsible for compliance with the statutory provisions of the data protection laws, in particular for the lawfulness of the transfer of data to the Processor as well as for the lawfulness of the data processing.
+
+The Processor is responsible for taking appropriate technical and organizational protection measures so that its processing complies with the legal requirements and ensures the protection of the rights of the Data Subjects.
+
+## Obligations of the processor
+
+
+## Bound by directions
+
+The Processor processes personal data in accordance with its privacy policy (cf. [Privacy Policy](privacy-policy.md)) and on the documented directions of the Customer. The initial direction result from the Agreement. Subsequent instructions shall be given either in writing, whereby e-mail shall suffice, or orally with immediate written confirmation.
```

Comment:
What does `cf` stand for?

Reply:
"vgl." in English. From Latin "conferatur" :)